docs/learn/learn-proxies.md

```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
 ---
 import RPC from "./../../components/RPC-Connection"
 
-Polkadot provides a module that allows users to set proxy accounts to perform a limited number of
-actions on their behalf. Much like the Stash and Controller account relationship in
-[staking](learn-staking.md), proxies allow users to keep one account in cold storage and actively
-participate in the network with the weight of the tokens in that account.
-
-Check out our Polkadot YouTube video that explains what proxy accounts are:
+Much like controller accounts in
+[staking](learn-staking.md), proxies allow users to use an account (it can be a stash account in cold storage or another account in a hot wallet) less frequently but actively
+participate in the network with the weight of the tokens in that account. Proxies can be viewed as an "extreme" version of a controller account, i.e. proxies are allowed to perform a limited amount of actions related to specific [substrate pallets](https://docs.substrate.io/reference/frame-pallets/) on behalf of another account. Check out the video below about proxies.
 
 [![Proxy Accounts](https://img.youtube.com/vi/1tcygkq52tU/0.jpg)](https://www.youtube.com/watch?v=1tcygkq52tU)
 
+
+## Why use a Proxy?
+
+Proxies are great to use for specific purposes because they add in a layer of security. Rather than
+using funds in one sole account, smaller accounts with unique roles complete tasks for the main
+stash account. This drives attention away from the main account and to proxies.
```

Comment:
Part of this is attention, but the main idea here is that proxies can be "hotter" than the initial account (stash), which can be kept cold but the "weight" of the tokens can be still be used by the hotter accounts.

Reply:
Will add this in the text